streams/src/main/java/org/apache/kafka/streams/kstream/KGroupedStream.java

```diff
@@ -42,201 +42,202 @@
 public interface KGroupedStream<K, V> {
 
     /**
-     * Count the number of records in this stream by the grouped key.
-     * Records with {@code null} key or value are ignored.
-     * The result is written into a local {@link KeyValueStore} (which is basically an ever-updating materialized view).
-     * Furthermore, updates to the store are sent downstream into a {@link KTable} changelog stream.
-     * <p>
-     * Not all updates might get sent downstream, as an internal cache is used to deduplicate consecutive updates to
-     * the same key.
-     * The rate of propagated updates depends on your input data rate, the number of distinct keys, the number of
-     * parallel running Kafka Streams instances, and the {@link StreamsConfig configuration} parameters for
-     * {@link StreamsConfig#CACHE_MAX_BYTES_BUFFERING_CONFIG cache size}, and
-     * {@link StreamsConfig#COMMIT_INTERVAL_MS_CONFIG commit intervall}.
-     * <p>
-     * For failure and recovery the store will be backed by an internal changelog topic that will be created in Kafka.
-     * The changelog topic will be named "${applicationId}-${internalStoreName}-changelog", where "applicationId" is
-     * user-specified in {@link StreamsConfig} via parameter
-     * {@link StreamsConfig#APPLICATION_ID_CONFIG APPLICATION_ID_CONFIG}, "internalStoreName" is an internal name
-     * and "-changelog" is a fixed suffix.
-     * Note that the internal store name may not be queriable through Interactive Queries.
-     *
+     * Count the number of records in this stream by the grouped key. Records with {@code null} key
+     * or value are ignored. The result is written into a local {@link KeyValueStore} (which is
```

Comment:
Please update you IDE setting to avoid unnecessary reformatting.

We also apply a "start a new line for each new sentence" policy, because it minimizes PR diffs if we update the JavaDocs and thus makes reviewing easier.